adam/learner/pursuit.py

```diff
@@ -1205,3 +1211,27 @@ def _fallback_templates(
         ) in self._concept_to_hypotheses_and_scores.items():
             for (graph_pattern, score) in graph_patterns_to_scores.items():
                 yield (concept, graph_pattern, score)
+
+    def _match_template(
+        self,
+        *,
+        concept: Concept,
+        pattern: PerceptionGraphTemplate,
+        perception_graph: PerceptionGraph,
+    ) -> Optional[Tuple[PerceptionGraphPatternMatch, SemanticNode]]:
+        """
+        Try to match our model of the semantics to the perception graph
+        """
+        matcher = pattern.graph_pattern.matcher(
+            perception_graph,
+            match_mode=MatchMode.NON_OBJECT,
+            # debug_callback=self._debug_callback,
```

Comment:
Debugging line commented out; can probably just remove

Reply:
I'd prefer to leave it in, because we should be able to pass the `debug_callback` into this function (as the learner should have an optional slot for one) but the pursuit learner currently doesn't. Leaving it commented makes it easier for someone to manually enable this debugging feature if needed explicitly until it's properly supported in this learner.

Reply:
Perhaps we should track this missing feature in an issue referenced here to explain the commented line?